test/client/integration.test.ts

```diff
@@ -45,6 +48,36 @@ describe("Client Integration", function() {
         const apiPacker = CoreAPI.new(storagePacker, genesis, provider, signer);
         const apiSyncer = CoreAPI.new(storageSyncer, genesis, provider, signer);
 
+        // Setup pubkeys, state for packer & syncer
+        const tokenID = 1;
+        const initialBalance = USDT.fromHumanValue("100.12");
+
+        for (const user of group.userIterator()) {
+            const state = State.new(
+                user.pubkeyID,
+                tokenID,
+                initialBalance.l2Value,
+                0
+            );
+            // Setup packer L2 storage
+            await storagePacker.pubkey.update(
+                user.pubkeyID,
+                new Pubkey(user.pubkey)
+            );
+            await storagePacker.state.update(user.stateID, state);
+
+            // Setup syncer L1 pubkeys
+            await apiSyncer.contracts.blsAccountRegistry.register(user.pubkey);
```

Comment:
The reference to the syncer here might be confusing. 

In the test case, we assume the syncer is agnostic to the public key before sync.

```suggestion
            // Setup L1 pubkeys
            await contracts.blsAccountRegistry.register(user.pubkey);
```